sphinx/writers/latex.py

```diff
@@ -487,7 +487,7 @@ def __init__(self, document: nodes.document, builder: "LaTeXBuilder") -> None:
         self.first_document = 1
         self.this_is_the_title = 1
         self.literal_whitespace = 0
-        self.no_contractions = 0
+        self.no_contractions = 0  # not used
```

Comment:
I was wondering is this needed some deprecation and how actually achieve that?

Reply:
Please replace this member by property.
This is an example.
https://github.com/sphinx-doc/sphinx/blob/24908e0fa9868ea278db25161d6b51038f1db3ca/sphinx/writers/latex.py#L2300-L2304

Reply:
Done